### PR TITLE
Filter out IPv4 broadcast address for any monitoring interface

### PR DESCRIPTION
**Pull request by jimyang2008**

### CODE REVIEW

File: hook/DeviceHook.js [hunk 1]
```diff
@@ -63,10 +63,22 @@ class DeviceHook extends Hook {
...
+      log.warn("Invalid MAC address for process device update:", event);
       return;
     }
 
+    /*
+     * Filter out IPv4 broadcast address for any monitoring interface
+     */
+    if (ipv4Addr) {
+      let monInterfaces = sysManager.getMonitoringInterfaces();
+      let foundInterface = monInterfaces.find(e => e.bcast_address === ipv4Addr)
```
Comment (jasonlyc) [inline hook/DeviceHook.js:75]: const ip = require('ip');
......
monInterfaces.find(e => e.subnet && ip.cidrSubnet(e.subnet).broadcastAddress === ipv4Addr)

File: util/linux.js [hunk 2]
```diff
@@ -87,6 +87,11 @@ exports.gateway_ip_for = function(nic_name) {
   return trim_exec_async("ip r | grep " + nic_name + " | grep default | cut -d ' ' -f 3 | sed -n '1p'");
 };
 
+exports.bcast_address_for = function(nic_name) {
+  var cmd = "ifconfig " + nic_name + " 2> /dev/null | fgrep 'Bcast:' | awk '{print $3}' |awk -F: '{print $2}'";
```
Comment (jasonlyc) [inline util/linux.js:91]: No need to do this in case output formay of ifconfig may vary on different platforms

File: hook/DeviceHook.js (hunk 1 above)
Comment (jimyang2008) [reply hook/DeviceHook.js]: fixed as suggested

File: util/linux.js (hunk 2 above)
Comment (jimyang2008) [reply util/linux.js]: reverted as suggested

**Pull request merged**